test/evergreen.yml

```diff
@@ -2534,6 +2543,9 @@ tasks:
   - <<: *format-stress-test
     name: format-stress-test-4
     tags: ["stress-test-4"]
+  - <<: *format-abort-recovery-stress-test
+    name: format-abort-recovery-stress-test-1
+    tags: ["abort-recovery-stress-test-1"]
```

Comment:
Ditto. I don't think it's necessary to use anchor references and tags, as it's merely a single task.

Reply:
See 7d3b047af45405f8727de69a509c88344b9566b9